crypto/asn1/a_bitstr.c

```diff
@@ -142,6 +142,9 @@ ASN1_BIT_STRING *ossl_c2i_ASN1_BIT_STRING(ASN1_BIT_STRING **a,
  */
 int ASN1_BIT_STRING_set_bit(ASN1_BIT_STRING *a, int n, int value)
 {
+    if (n < 0)
```

Comment:
Code should be added after declarations, no ?

Reply:
Yes, I just made the necessary changes